components/Form/Form.js

```diff
@@ -0,0 +1,185 @@
+import { useState } from "react";
+import {
+  Form,
+  Fieldset,
+  Legend,
+  Label,
+  Input,
+  Textarea,
+  Select,
+  Checkbox,
+  FormButton,
+  SomeoneElse,
+} from "./Form.styled";
+import PlanList from "../PlanList/PlanList";
+export default function Plan() {
+  const [plan, setPlan] = useState([]);
+  const [forSomeoneElse, setForSomeoneElse] = useState(false);
+  const handleForChange = (event) => {
+    setForSomeoneElse(event.target.value === "Someone else");
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const frequency = [];
+    for (let [key, value] of formData.entries()) {
```

Comment:
Why are you using an array to store the different frequency parameters? Aren't these two separate aspects of the frequency? The first one could be sth. like `interval` and the second one could be called `frequency`. I would store them under separate names instead of an array, because in an array you always have to remember, which index represents which variable.